README.md

```diff
@@ -54,7 +58,8 @@ F:\conda-envs\cellpose-omnipose-biop-gpu\python.exe
 ```
 
 > **Note**
-> While this example is done under Windows, this will work on Mac and Linux as well. 
+> While this example is done under Windows, this will work on Mac and Linux as well. You may need to change
+> `where` to `which`.
```

Comment:
We can add explicit examples for both Win and Mac/Linux so that people see examples of what it looks like

Reply:
Sure. 
I just tested on my Mac and my linux VM (ubuntu 23.10) in Bash shell `where` doesn't work. 
In `zsh` the default shell on macOS now, `where` does work. Default Ubuntu doesn't have zsh.
`which` works in both shells and both platforms.
I'll edit this section shortly.